File: trunk/dobson/DobsonLibrary/Domain/Context.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DobsonLibrary.Domain
{
    public class Context
    {

        public int contextID { get; set; }
        public string contextName { get; set; }
        
        public Context(int id, string name)
        {
            contextID = id;
            contextName = name;
        }
        public Context()
        {
            contextID = -1;
            contextName = "";
        }

    }
}

File: trunk/OWLMorpheus/src/db/Context.java
package db;

public class Context {
	private int ContextId;
	private String ContextName;
	
	public Context(int id, String descriptor) {
		this.ContextId = id;
		this.ContextName = descriptor;
	}
	
	public int getContextId() {
		return ContextId;
	}
	
	public String getContextName() {
		return ContextName.replaceAll(" ", "_");
		
	}
}
